src/iron_math.py

```diff
@@ -1,16 +1,35 @@
 import math
+import arcade
 from typing import Tuple
 
 
-def rotate_vec2(point: Tuple[float, float], radians: float, /) -> float:
+def rotate_vec(point: Tuple[float, float], radians: float, /) -> float:
     x, y = point
     s = math.sin(radians)
     c = math.cos(radians)
     return (c * x - s * y, s * x + c * y)
 
 
-def add_vec2(point: Tuple[float, float], translation: Tuple[float, float], /) -> float:
+def add_vec(point: Tuple[float, float], translation: Tuple[float, float], /) -> float:
     return (point[0] + translation[0], point[1] + translation[1])
 
-def scale_vec2(point: Tuple[float, float], factor: float, /) -> float:
+
+def scale_vec(point: Tuple[float, float], factor: float, /) -> float:
     return (point[0] * factor, point[1] * factor)
+
+
+def offset_sprite_from(
```

Comment:
I think this fcn could use some clean-up.  It is not clear to me what this fcn does, nor what the offset argument represents.

Reply:
We can add some docstrings to these functions.  The term "offset" can be replaced with "transform," though I think I remember objections to the term "transform," can't remember from where at the moment.

Reply:
Yeah, if it makes it more clear, we can replace the term "offset" with "transform" everywhere